e2e/playwright/share-link.spec.ts

```diff
@@ -1,45 +1,51 @@
 import { expect, test } from '@e2e/playwright/zoo-test'
+import type { Page } from '@playwright/test'
+
+async function navigateAndClickOpenInDesktopApp(
+  page: Page,
+  codeLength: number
+) {
+  const code = Array(codeLength).fill('0').join('')
+  const targetURL = `?create-file=true&browser=test&code=${code}&ask-open-desktop=true`
+  expect(targetURL.length).toEqual(codeLength + 58)
+  await page.goto(page.url() + targetURL)
+  expect(page.url()).toContain(targetURL)
+  const button = page.getByRole('button', { name: 'Open in desktop app' })
+  await button.click()
+}
+
+function getToastError(page: Page) {
+  return page.getByText('The URL is too long to open in the desktop app')
+}
 
-const isWindows =
-  navigator.platform === 'Windows' || navigator.platform === 'Win32'
 test.describe('Share link tests', () => {
-  ;[
-    {
-      codeLength: 1000,
-      showsErrorOnWindows: false,
-    },
-    {
-      codeLength: 2000,
-      showsErrorOnWindows: true,
-    },
-  ].forEach(({ codeLength, showsErrorOnWindows }) => {
-    test(
-      `Open in desktop app with ${codeLength}-long code ${isWindows && showsErrorOnWindows ? 'shows error' : "doesn't show error"}`,
-      { tag: ['@web'] },
-      async ({ page }) => {
-        if (process.env.TARGET !== 'web') {
-          // This test is web-only
-          // TODO: re-enable on CI as part of a new @web test suite
-          return
-        }
+  test(
+    `Open in desktop app with 2000-long code doesn't show error on non-Windows`,
+    { tag: ['@web', '@macos', '@linux'] },
+    async ({ page }) => {
+      if (process.env.TARGET !== 'web' || process.platform === 'win32') {
+        return
```

Comment:
What do you think about calling `test.skip()` instead? I think that may present better when viewing the history of results.